pysolr.py

```diff
@@ -9,7 +13,6 @@
 import re
 import time
 from xml.etree import ElementTree
-from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
This appears to be an inadvertent regression – it would probably be easiest to rebase your branch against the current master & re-push it to update the pull request.